salt/haproxy/init.sls

```diff
@@ -0,0 +1,29 @@
+---
+include:
+  - repositories
+
+haproxy:
+  pkg.installed:
+  - pkgs:
+    - kubernetes-node
+  - require:
+    - file: /etc/zypp/repos.d/containers.repo
```

Comment:
Is this still necessary? It feels odd to be asking for kubernetes-node to be installed here